app/views/relationships/index.html.haml

```diff
@@ -0,0 +1,24 @@
+%section.people
+  %header
+    %h2 People I Follow
+  %table.table
+    %thead
+      %tr
+        %th(width="30%")
+        %th(width="20%") Videos in Queue
+        %th(width="20%") Followers
+        %th(width="30%") Unfollow
+    %tbody
+      -@relationships.each do |relationship|
+        %tr
+          %td
+            %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(relationship.leader.email.downcase)}?s=40")
```

Comment:
I would refactor this in 2 ways:

1. Use `= image_tag ...` helper
2. Create a helper method `gravatar_for` that would be in `ApplicationHelper` and would accept user as argument. The method would wrap that long url.